src/relations.jl

```diff
@@ -344,6 +345,33 @@ function air_temperature(
     ) / cv_m(param_set, q)
 end
 
+
+## YAIR check this
+"""
+    air_temperature(param_set, h, q::PhasePartition)
+
+The air temperature, where
+
+ - `param_set` an `AbstractParameterSet`, see the [`Thermodynamics`](@ref) for more details
+ - `h` internal energy per unit mass
+and, optionally,
+ - `q` [`PhasePartition`](@ref). Without this argument, the results are for dry air.
+"""
+function air_temperature_from_enthalpy(
+    param_set::APS,
+    h::FT,
+    q::PhasePartition{FT} = q_pt_0(FT),
+) where {FT <: Real}
+    cp_m_ = cp_m(param_set, q)
+    T_0::FT = ICP.T_0(param_set)
+    R_d::FT = ICP.R_d(param_set)
+    LH_v0::FT = ICP.LH_v0(param_set)
+    LH_f0::FT = ICP.LH_f0(param_set)
+    e_int_i0::FT = ICP.e_int_i0(param_set)
+    q_vap::FT = vapor_specific_humidity(q)
+    return (h + cp_m_*T_0 - q_vap*LH_v0} + q_i*LH_f0 - (FT(1-q.tot)*R_d*T_0)/cp_m_
```

Comment:
```suggestion
    return (h + cp_m_*T_0 - q_vap*LH_v0) + q_i*LH_f0 - ((1-q.tot)*R_d*T_0)/cp_m_
```
Is the `}` supposed to be `)`?